src/java/install.sh

```diff
@@ -206,15 +206,15 @@ sdk_install() {
     local full_version_check=${5:-".*-[a-z]+"}
     local set_as_default=${6:-"true"}
     if [ "${requested_version}" = "none" ]; then return; fi
-    # Blank will install latest stable version SDKMAN has
-    if [ "${requested_version}" = "latest" ] || [ "${requested_version}" = "lts" ] || [ "${requested_version}" = "default" ]; then
+     # Blank will install latest stable version SDKMAN has
+    if [ "${requested_version}" = "default" ]; then
          requested_version=""
     elif echo "${requested_version}" | grep -oE "${full_version_check}" > /dev/null 2>&1; then
         echo "${requested_version}"
     else
         local regex="${prefix}\\K[0-9]+\\.?[0-9]*\\.?[0-9]*${suffix}"
         local version_list=$(su ${USERNAME} -c ". \${SDKMAN_DIR}/bin/sdkman-init.sh && sdk list ${install_type} 2>&1 | grep -oP \"${regex}\" | tr -d ' ' | sort -rV")
-        if [ "${requested_version}" = "latest" ] || [ "${requested_version}" = "current" ]; then
+        if [ "${requested_version}" = "latest" ] || [ "${requested_version}" = "current" ] || [ "${requested_version}" = "lts" ] || [ "${requested_version}" = "latest" ]; then
```

Comment:
How is this condition block finding `lts` version?

Reply:
earlier, whenever it would find `lts` or `latest` as the required version, then it would simply skip the finding of version from this condition block and would go on installing `tem` instead of `ms`. So, checking for `lts` or `latest` and then entering this code block, makes consider anything coming out of `get_jdk_distros` function, where the execution checks in earlier than the `sdk_install` function

Reply:
> requested_version="$(echo "${version_list}" | head -n 1)"

I am talking about ^

LTS is not necessarily the latest or current version. Hence, the way this PR is now finding the LTS version is not correct. 

Do we have a way to determine the lts version for the given distro?

Reply:
corrected logic for lts finding